packages/flutter/lib/src/widgets/text.dart

```diff
@@ -661,6 +662,7 @@ class Text extends StatelessWidget {
       result = MouseRegion(
         cursor: DefaultSelectionStyle.of(context).mouseCursor ?? SystemMouseCursors.text,
         child: _SelectableTextContainer(
+          selectableId: key,
```

Comment:
Is this ok even though it defaults to null?

Reply:
What do you mean by ok? If the user does not provide a `key` then they will not be able to tie their state by to the `SelectedContentRange` the `SelectionListener` returns, unless the `SelectionListener` is only wrapping a single `Text` widget.